source/Tutorials/First-Time-Building.rst

```diff
@@ -1,6 +1,102 @@
+.. _First-Time-Building-Space-ROS:
+
 First Time Building Space ROS
 =============================
 
-TODO:
+.. contents:: Table of Contents
+   :depth: 1
+   :local:
+
+**Goal:** Create a workspace using Earthly for development and testing.
+
+**Tutorial level:** Beginner
+
+**Time:** 10 minutes (not including build time)
+
+Prerequisites
+-------------
+
+* `git installation <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`__
+* `earthly installation <https://earthly.dev/get-earthly>`__ which requires `Docker <https://docker.com>`__ (recommended) or `Podman <https://podman.io>`__.
+* Understanding of basic terminal commands (`here’s a guide for Linux <http://www.ee.surrey.ac.uk/Teaching/Unix/>`__)
+
+Introduction
+------------
+
+Space ROS, like ROS 2, is composed of a number of independent software projects each maintained in its own repositories.
+The ROS community has a number of tools to help organize and improve working with multiple repositories.
+Earthly is an automation tool where each target is run in a dedicated container with the ability to share artifacts between targets and the local host.
+The Space ROS project primarily uses earthly to create automation that runs consistently in local development, and in hosted infrastructure such as GitHub Actions.
```

Comment:
earthly -> "Earthly"

Reply:
Additional occurrences below in the text. 

Should consistently use "Earthly" when referring to the name of the tool in the abstract. "earthly" is OK when referring to the name of the binary.  This is my assumption, at least. I recommend following the conventions used in the Earthly docs.

Reply:
Yeah I agree that it should be consistent. Corrected in f4f1277a1d7573eeff21134b6fda5807ec1c489b